recipe/meta.yaml

```diff
@@ -31,37 +31,36 @@ build:
     - rd /s /q %SRC_DIR%\bootloader\waflib  # [win]
     # Build the bootloader
     - pushd bootloader
-    - waf --prefix="${PREFIX}" --target-arch=64bit --clang distclean all  # [osx]
-    - waf --prefix="${PREFIX}" --gcc --no-lsb distclean all     # [linux]
+    - waf --prefix="${PREFIX}" --clang distclean all  # [osx]
+    - waf --prefix="${PREFIX}" --gcc distclean all     # [linux]
     - waf --prefix="%LIBRARY_PREFIX%" distclean all             # [win]
     - popd
-    - "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv"
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   preserve_egg_dir: True
-  overlinking_ignore_patterns:   # [s390x]
-    - '*'                        # [s390x]
 
 requirements:
   build:
     - {{ compiler("c") }}
-
+    - patch           # [not win]
+    - m2-patch        # [win]
   host:
     - pip
     - python
     - setuptools
     - wheel
     - waf
-
+    - zlib
   run:
+    - python
+    - setuptools >=42.0.0
     - altgraph
     - binutils  # [aarch64]
-    - python
     - pefile >=2022.5.30  # [win]
-    - pyinstaller-hooks-contrib >=2021.4
-    - pywin32  # [win]
-    - pywin32-ctypes  # [win]
+    - pywin32-ctypes >=0.2.1 # [win]
     - macholib >=1.8  # [osx]
-    - setuptools >=42.0.0  # due to pkg_resources import
-    - importlib-metadata >=1.4  # [py<38]
+    - pyinstaller-hooks-contrib >=2024.7
+    - importlib-metadata >=4.6  # [py<310]
+    - packaging >=22.0
 
 test:
   imports:
```

Comment:
The license should be `GPL-2.0-or-later WITH Bootloader-exception AND Apache-2.0`, see https://github.com/pyinstaller/pyinstaller/blob/develop/COPYING.txt#L104 and https://github.com/pyinstaller/pyinstaller/blob/develop/COPYING.txt#L118

Reply:
Nice catch, thank you! Should the `license_family` remain `GPL` or should it be changed to `Other`?

Reply:
Thanks y'all!